src/kyc.ts

```diff
@@ -56,10 +91,40 @@ export type PersonalDataAndDocumentsKyc = z.infer<
   typeof personalDataAndDocumentsKycSchema
 >
 
+export const personalDataAndDocumentsDetailedKycSchema =
+  personalDataAndDocumentsKycSchema
+    .omit({ identificationDocument: true })
+    .and(
+      z.object({
+        email: z.string().regex(EMAIL_REGEX),
+        identificationDocumentFront: z.string(),
+        identificationDocumentType: identificationDocumentTypeSchema,
```

Comment:
Actually, is this line required, considering both of the options below specify a more narrowly typed version of the same field?

Reply:
yeah I'll just get rid of it